File: c-sharp/sem01/j05/sql_backend/Model/Capteur.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
// Représente un objet métier (juste des données).

namespace sql_backend.Model
{
    public class Capteur
    {
        public int Id { get; set; }
        public string Nom { get; set; }
        public string Type { get; set; }
        public string Unite { get; set; }
        public void display() {
            Console.WriteLine($"Id: {Id} ; Nom: {Nom} ; Type: {Type} ; Unite: {Unite}");
            }
    }

}

File: java/sem01/j05/sql_backend/src/model/Capteur.java
// Représente un objet métier (juste des données).

package model;

public class Capteur
    {
        private int id;
        private String nom, type, unite;

        public int getId() { return id; }
        public void setId(int id) { this.id = id; }

        public String getNom() { return nom; }
        public void setNom(String nom) { this.nom = nom; }

        public String getType() { return type; }
        public void setType(String type) { this.type = type; }

        public String getUnite() { return unite; }
        public void setUnite(String unite) { this.unite = unite; }

        public void display() {
            System.out.println("Id: " + id + " ; Nom: " + nom + " ; Type: " + type + " ; Unite: " + unite);
        }
    }